### PR TITLE
Feat/paging flight

### PR DESCRIPTION
**Pull request by nepgya2803**

### CODE REVIEW

File: TicketBooking.Common/AppExceptions/RequestParam.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿namespace TicketBooking.Service.Models
+{
+    public class RequestParam
```
Comment (tanngocIT) [inline TicketBooking.Common/AppExceptions/RequestParam.cs]: The name doesn't make sense for pagination.

File: TicketBooking.Service/Services/FlightService/FlightService.cs [hunk 2]
```diff
@@ -24,22 +18,23 @@ public class FlightService : IFlightService
         private readonly IAircraftRepository aircraftRepo;
         private readonly IAirportService airportService;
         private readonly IFlightScheServices flightScheServices;
-        private readonly IFlightScheRepository flightScheRepo;
-        private readonly ILogger<Flight> logger;
+        // private readonly IFlightScheRepository flightScheRepo;
+        // private readonly ILogger<Flight> logger;
```
Comment (tanngocIT) [inline TicketBooking.Service/Services/FlightService/FlightService.cs]: remove comment

File: TicketBookingAPI/Controllers/FlightController.cs [hunk 3]
```diff
@@ -99,6 +99,18 @@ public async Task<IActionResult> GetFlightByRequest([FromQuery] FlightRequest fl
             
             return Ok(await flightservice.GetFlightAsync(flightrequest));
         }
+        
+        [HttpGet("GetflightByRequestWithPaging")]
```
Comment (tanngocIT) [inline TicketBookingAPI/Controllers/FlightController.cs]: naming convention..............

File: TicketBookingAPI/Controllers/FlightController.cs [hunk 4]
```diff
@@ -99,6 +99,18 @@ public async Task<IActionResult> GetFlightByRequest([FromQuery] FlightRequest fl
...
+        public async Task<IActionResult> GetFlightByRequestPaging([FromQuery] FlightRequest flightrequest, [FromQuery] RequestParam request)
```
Comment (tanngocIT) [inline TicketBookingAPI/Controllers/FlightController.cs]: should combine FlightRequest and RequestParam into one model. Use inherit

File: TicketBookingAPI/Controllers/FlightController.cs (hunk 3 above)
Comment (nepgya2803) [reply TicketBookingAPI/Controllers/FlightController.cs]: fixed

File: TicketBooking.Service/Services/FlightService/FlightService.cs (hunk 2 above)
Comment (nepgya2803) [reply TicketBooking.Service/Services/FlightService/FlightService.cs]: fixed

File: TicketBooking.Common/AppExceptions/RequestParam.cs (hunk 1 above)
Comment (nepgya2803) [reply TicketBooking.Common/AppExceptions/RequestParam.cs]: fixed

File: TicketBookingAPI/Controllers/FlightController.cs (hunk 4 above)
Comment (nepgya2803) [reply TicketBookingAPI/Controllers/FlightController.cs]: fixed

**Review by tanngocIT — APPROVED** [review]

**Pull request merged**